SPECS/ruby/ruby.spec

```diff
@@ -62,6 +62,9 @@ sudo -u test make test TESTS="-v"
 %{_mandir}/man5/*
 
 %changelog
+* Thu Jan 20 2022 Cameron Baird <cameronbaird@microsoft.com> - 2.6.7-3
+- Filter out references to module_info.ld in pkgconfigs
```

Comment:
I would suggest a slightly different description to clarify that the package release is increased to republish the package even though no changes are made to this spec. (applies to all 6 specs)
"Bump release to build and republish with mariner-rpm-macros fix to filter out references to module_info.ld in pkgconfig files"